tests/test_card_tables.py

```
@@ -0,0 +1,207 @@
+#!/usr/bin/env python3
+"""
+Test script for Card Table functionality in Basecamp MCP integration.
+"""
+
+import json
```

Flake8 (7.2.0)

[error] 'json' imported but unused

(F401)